### PR TITLE
IPVS: Add support for GetConfig/SetConfig

### PR DESCRIPTION
**Pull request by lbernail**
GetConfig and SetConfig allow to change the IPVS connection timeouts for TCP, TCPFIN and UDP

We vendor libnetwork's ipvs library into kubernetes and we would like to change the default IPVS timeouts (and probably make them configurable).

It's my first PR in this repository, so of course don't hesitate to let me know if things should be done differently. I'm also happy to create an issue if necessary.

### CODE REVIEW

File: ipvs/netlink.go [hunk 1]
```diff
@@ -503,6 +504,60 @@ func (i *Handle) doGetDestinationsCmd(s *Service, d *Destination) ([]*Destinatio
...
+	}
+	return res, nil
+}
+
+// doSetConfigCmd a wrapper function to be used by SetConfig
+func (i *Handle) doSetConfigCmd(c *Config) error {
+	req := newIPVSRequest(ipvsCmdSetConfig)
+	req.Seq = atomic.AddUint32(&i.seq, 1)
+
+	req.AddData(nl.NewRtAttr(ipvsCmdAttrTimeoutTCP, nl.Uint32Attr(uint32(c.TimeoutTCP.Seconds()))))
```
Comment (fcrisciani) [inline ipvs/netlink.go]: isn't it better to make them conditional? what is the config does not specify all the 3 values? is the 0 value ignored by the kernel?

Comment (lbernail) [reply ipvs/netlink.go]: Yes, 0 means "unchanged"
Extract from the ipvsdam man page:
```
--set tcp tcpfin udp
Change the timeout values used for IPVS connections. This command always takes 3 parameters, representing the timeout values (in seconds) for TCP sessions, TCP sessions after receiving a FIN packet, and UDP packets, respectively. A timeout value 0 means that the current timeout value of the corresponding entry is preserved.
```
But I can definitely add a test for this

Comment (fcrisciani) [reply ipvs/netlink.go]: that would be great and maybe a comment saying that 0 means unchanged just for posterity, rest LGTM

Comment (lbernail) [reply ipvs/netlink.go]: I'm doing it right now

**Comment by fcrisciani** [comment]
thanks @lbernail merged!

**Comment by lbernail** [comment]
That was very fast. Thanks a lot!

**Pull request merged**